neo4j/bookmarks.go

```diff
@@ -61,24 +64,24 @@ type BookmarkManagerConfig struct {
 	// Hook called whenever bookmarks for a given database get updated
```

Comment:
Cool idea!

Question: if the notifier or supplier timeout (due to respecting the provided context), how should they communicate that back to the BMM/driver? They can't return an error and abort the current driver action, can they?

Reply:
I thought about this and I am torn.

On one hand, I could expose `error` in the return type to the user callback signatures. 
The problem then becomes: what do I do with the returned error?
Is it worth forwarding the error and aborting the operation for that?

Note: in the built-in implementation, the only sources for `error` are the consumer callbacks.

Reply:
Let me give it a try

Reply:
fixed in 4087bd18ba694bb0a9f4ead5652e10ac5c327f84